Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Corefinement/face_graph_utils.h

```diff
@@ -1177,6 +1204,8 @@ void append_patches_to_triangle_mesh(
     }
   }
 
+    user_visitor.build_output_step();
```

Comment:
```suggestion
    user_visitor.filter_coplanar_patch_interior_edges();
```

Reply:
Is that a proposal for a better name??

Reply:
I thought you wanted me to propose names for each step